dags/solutions_team/solutionsteam_tf_release_se_supported.py

```diff
@@ -18,17 +18,19 @@
 from airflow import models
 from dags import composer_env
 from dags.vm_resource import TpuVersion, Zone, RuntimeVersion, V5_NETWORKS, V5E_SUBNETWORKS, V5P_SUBNETWORKS
-from dags.solutions_team.configs.tensorflow import solutionsteam_tf_2_16_supported_config as tf_config
+from dags.solutions_team.configs.tensorflow import solutionsteam_tf_release_supported_config as tf_config
 from dags.solutions_team.configs.tensorflow import common
 
 
 # Release tests only need to run once, they can be run manually as needed
 SCHEDULED_TIME = None
+VERSION = f"{tf_config.MAJOR_VERSION}.{tf_config.MINOR_VERSION}"
+
 
 with models.DAG(
-    dag_id="tf_2_16_se_nightly_supported",
+    dag_id=f"tf_{tf_config.MAJOR_VERSION}_{tf_config.MINOR_VERSION}_se_nightly_supported",
```

Comment:
I think major version and other variables could be defined in the dag level. In this case:

* 2.16 dag could pass in 2.16 as major version;
* 2.15.1 dag could pass in 2.15 as major version, and 1 as minor version

HDYT?

Reply:
That would affect non-tf DAGS though which is a pretty big increase in scope for this change. I'm not sure I want to add it here?

I agree that being able to specify the version in the dag itself could be useful though! But perhaps as a future change.

Reply:
I think we'd only ever want to have a single release in flight at any given time -- that's what we do today anyway.

Although I could see a world where we leave release DAGs in place for future reference. I don't think it's a bad refactor regardless, it separates the logic of how to build a release DAG from which release to build a DAG for.

Reply:
Spoke to Ran offline and we're going to add this as a future change. Resolving for now.